opal/mca/event/libevent2022/configure.m4

```diff
@@ -159,8 +159,13 @@ AC_DEFUN([MCA_opal_event_libevent2022_CONFIG],[
 
     AC_MSG_RESULT([$event_args])
 
+    # We added this in order to enforce the use of internal random
+    # function to prevent pertubing the random seed of the user.
+    # libevent uses random function in poll(), select() and evutil().
```

Comment:
Slightly tweak the wording:

> We define "random" to be "opal_random" so that Libevent will not use random(3) internally (and potentially unexpectedly perturb values returned by rand(3) to the application).